src/css/mixins.scss

```diff
@@ -2,99 +2,87 @@
   height: 100%;
   width: 100%;
 }
+
+@mixin default-cards() {
+  display: grid;
+  grid-gap: 30px;
+  padding: 30px 0 30px 30px;
+  width: 95.5vw;
+}
+
+@mixin small-screen-header() {
+  h1 {
+    width: 100%;
+  }
+  .user-info {
+    width: 100%;
+  }
+  .search-bar {
+    width: 100%;
+  }
+}
+
 @mixin xs {
-   @media (min-width: #{$screen-xs}) {
-       @content;
-       h1 {
-         width: 100%;
-       }
-       .user-info {
-         width: 100%;
-       }
-       .search-bar {
-         width: 100%;
-       }
-       .all-cards {
-         display: grid;
-         grid-gap: 30px;
-         grid-template-columns: repeat(1, 1fr);
-         padding: 20px 0 20px 20px;
-         width: 95.5vw;
-       }
-       .main-header {
-         flex-flow: row wrap;
-       }
-   }
+  @media (min-width: #{$screen-xs}) {
+    @content;
+    @include small-screen-header;
+    .all-cards {
+      @include default-cards;
+      grid-template-columns: repeat(1, 1fr);
+    }
+    .main-header {
+      flex-flow: row wrap;
+    }
+  }
 }
+
 @mixin sm {
-   @media (min-width: #{$screen-sm}) {
-       @content;
-       h1 {
-         width: 100%;
-       }
-       .user-info {
-         width: 100%;
-       }
-       .search-bar {
-         width: 100%;
-       }
-       .all-cards {
-         display: grid;
-         grid-gap: 30px;
-         grid-template-columns: repeat(2, 1fr);
-         padding: 20px 0 20px 20px;
-         width: 95.5vw;
-       }
-   }
+  @media (min-width: #{$screen-sm}) {
```

Comment:
All of these @media mixins look so much more concise and cleaned up compared to the previous version.  Nice work!